tailwind.config.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    darkMode: ["class"],
+    content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+    "./src/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+  	extend: {
+  		borderRadius: {
+  			lg: 'var(--radius)',
```

Comment:
Curious why you added the borderRadius theming?

Reply:
Just playing around, no particular reason. I removed it.

Reply:
I have been watching some videos on tailwind. I had no idea what it was until you mentioned it.